tests/integration/cgroups.bats

```diff
@@ -70,5 +70,11 @@ EOF
 
     # update kernel memory limit
     run "$RUNC" update test_cgroups_kmem --kernel-memory 50331648
-    [ ! "$status" -eq 0 ]
+    # Since kernel 4.6, we can update kernel memory without initialization
+    # because it's accounted by default.
+    if [ "$KERNEL_MAJOR" -lt 4 ] || [ "$KERNEL_MAJOR" -eq 4 -a "$KERNEL_MINOR" -le 5 ]; then
+        [ ! "$status" -eq 0 ]
+    else
+        [ "$status" -eq 0 ]
```

Comment:
Can we also check that the limit was set properly? And personally I'd prefer to make it have two pass cases:

1. It fails. (we can check that the kernel version is right here).
2. It succeeds and the limit is set properly.

Also, the kernel version check looks wrong. `4.3.3` would fail to be considered as an "old" kernel.